gG3TT3Rr/HW7/hw3.py

```diff
@@ -0,0 +1,11 @@
+def count_chars(s):
```

Comment:
good job
look at this implementation:
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result